SCClassLibrary/Common/Control/Buffer.sc

```diff
@@ -31,11 +31,13 @@ Buffer {
 	}
 
 	*allocConsecutive { arg numBufs = 1, server, numFrames, numChannels = 1, completionMessage, bufnum;
-		var	bufBase, newBuf;
+		var	bufBase;
 		bufBase = bufnum ?? { server.nextBufferNumber(numBufs) };
 		^Array.fill(numBufs, { |i|
-			newBuf = Buffer.new(server, numFrames, numChannels, i + bufBase);
-			server.sendMsg(\b_alloc, i + bufBase, numFrames, numChannels,
+			// note, cannot use alloc or allocMsg here
+			// because those methods don't pass a buffer index for completion message
+			var newBuf = Buffer.new(server, numFrames.asInteger, numChannels.asInteger, i + bufBase);
```

Comment:
it would save a little bit of electricity if you converted `numFrames` and `numChannels` to integer outside the fill-loop once?

Reply:
> it would save a little bit of electricity if you converted numFrames and numChannels to integer outside the fill-loop once?

OK, done.

Reply:
> it would save a little bit of electricity if you converted numFrames and numChannels to integer outside the fill-loop once?

OK, done.